### PR TITLE
vigr: add page

### PR DESCRIPTION
**Pull request by F-Sossi**
Added vigr command for modifying the /etc/group file

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/vigr.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# vigr
+
+> Edit the group file.
+> More information: <https://manned.org/vigr>.
+
+- Edit the group file:
+
+`vigr`
+
+- Display the current version of `vigr`:
```
Comment (navarroaxel) [inline pages/linux/vigr.md:10]: The examples are about `vigr` so
```suggestion
- Display version:
```

**Review by navarroaxel — COMMENTED** [review]
Hi @F-Sossi, welcome and thanks for your submission. I left a comment

**Review by CleanMachine1 — APPROVED** [review]
Once axel's comment is applied

File: pages/linux/vigr.md (hunk 1 above)
Comment (F-Sossi) [reply pages/linux/vigr.md]: Thanks for the feedback love this project.

**Review by mfrw — APPROVED** [review]
LGTM :)

Thank you @F-Sossi for your contribution 🥇

**Pull request merged**